site/docs/v1/tech/oauth/endpoints.adoc

```diff
@@ -934,6 +934,17 @@ Ensure you are sending these parameters as form encoded data in the request body
 Do not send these parameters in a query string or as JSON.
```

Comment:
We should also probably modify the "You must provide client authentication using the Basic Authentication Scheme authorization header." verbiage in the request header section too.